subsys/net/l2/ethernet/ethernet.c

```diff
@@ -68,21 +68,31 @@ void net_eth_ipv6_mcast_to_mac_addr(const struct in6_addr *ipv6_addr,
 	}
 
 #ifdef CONFIG_NET_VLAN
-#define print_vlan_ll_addrs(pkt, type, tci, len, src, dst)		   \
+#define print_vlan_ll_addrs(pkt, type, tci, len, src, dst, tagstrip)       \
 	if (CONFIG_NET_L2_ETHERNET_LOG_LEVEL >= LOG_LEVEL_DBG) {	   \
 		char out[sizeof("xx:xx:xx:xx:xx:xx")];			   \
 									   \
 		snprintk(out, sizeof(out), "%s",			   \
 			 net_sprint_ll_addr((src)->addr,		   \
 					    sizeof(struct net_eth_addr))); \
 									   \
-		NET_DBG("iface %p src %s dst %s type 0x%x tag %d pri %d "  \
-			"len %zu",					   \
-			net_pkt_iface(pkt), log_strdup(out),		   \
-			log_strdup(net_sprint_ll_addr((dst)->addr,	   \
-					    sizeof(struct net_eth_addr))), \
-			type, net_eth_vlan_get_vid(tci),		   \
-			net_eth_vlan_get_pcp(tci), (size_t)len);	   \
+		if (tagstrip) {						   \
```

Comment:
I think we can optimize this a bit more so instead of this `if() else` we can do it like this:
```
	NET_DBG("iface %p src %s dst %s type 0x%x "        \
		"tag %d %spri %d len %zu",        \
		net_pkt_iface(pkt), log_strdup(out),	   \
		log_strdup(net_sprint_ll_addr((dst)->addr, \
			   sizeof(struct net_eth_addr))),  \
		type, net_eth_vlan_get_vid(tci),	   \
		tagstrip ? "(stripped) " : "",		   \
		net_eth_vlan_get_pcp(tci), (size_t)len);   \
```

Reply:
Yes, modified now.